src/rgw/driver/rados/rgw_sync.cc

```diff
@@ -1647,6 +1647,95 @@ class RGWMetaSyncShardCR : public RGWCoroutine {
         collect_children();
       } while (omapkeys->more && can_adjust_marker);
 
+      mdlog_marker = sync_marker.marker;
+      marker = max_marker = sync_marker.marker;
+      do {
+#define INCREMENTAL_MAX_ENTRIES 100
```

Comment:
let's not copy-paste any #define magic numbers code, even though some older code does this.